pkg/splunk/common/urls_testingcode.go

```diff
@@ -93,62 +81,18 @@ const (
 	TestURLPeerHeadlessDecommission = "https://splunk-stack1-indexer-0.splunk-stack1-indexer-headless.test.svc.cluster.local:8089" + URIPeerDecommission
 )
 
-// ***** License Manager *****
```

Comment:
By switching the testing code to `*Manager` we no longer need these exceptions. 
In the coming sprints, we will also review APIs that ES now supports with `*Manager`